hapi-utils/src/main/java/com/hederahashgraph/fee/FeeBuilder.java

```diff
@@ -295,8 +295,8 @@ public static int getSignatureSize(Transaction transaction) {
    * @return tinyHbars
    */
   public static long getTinybarsFromTinyCents(ExchangeRate exchangeRate, long tinyCentsFee) {
-    BigInteger hbarMultiplier = BigInteger.valueOf(Long.valueOf(exchangeRate.getHbarEquiv()));
-    BigInteger centsDivisor = BigInteger.valueOf(Long.valueOf(exchangeRate.getCentEquiv()));
+    BigInteger hbarMultiplier = BigInteger.valueOf(exchangeRate.getHbarEquiv());
+    BigInteger centsDivisor = BigInteger.valueOf(exchangeRate.getCentEquiv());
```

Comment:
I don't see any problem with the original code, because `getHbarEquiv` and `getCentEquiv` return an `int`. If sonar is complaining, that really is their problem! We can do this code differently by multiplying with `1L` or do a cast, but I don't see the point.

Reply:
Yes , made the change as SonarCloud had complained about casting to `Long`. Reverted back the change.

Reply:
As discussed, I just wanted to point out that this implicit cast is done with courtesy for us. So if we do it explicitly to remove the dependency on this courtesy, there should not be any complaint! But I am ok to remove it.